axelrod/history.py

```diff
@@ -33,6 +33,8 @@ def __init__(self, plays=None, coplays=None):
     def append(self, play, coplay):
         """Appends a new (play, coplay) pair an updates metadata for
         number of cooperations and defections, and the state distribution."""
+        # casts plays sent as integers into Action objects
+        play, coplay = Action(play), Action(coplay)
```

Comment:
This feels unnecessary to me, which strategies are not returning proper `Action`s?